src/chimera/interfaces/telescope.py

```diff
@@ -417,6 +417,28 @@ def unparkComplete(self):
         successfully.
         """
 
+    def openCover(self):
```

Comment:
Tiago,

you are right. There are two systems in place to check for capabilities,
focuser and camera uses a capability check while telescope was separated in
interfaces basically because otherwise you'll end with a huge interfaces.

Telescope users are supposed to test if the scope support a particular
interface, I think today we assume it does implement everything, which is
not right. We might have to fix this, but given that a single interface
would be giant I think we should stay this way.

Cheers,

2015-10-23 15:09 GMT-02:00 Tiago <notifications@github.com>:

> In src/chimera/interfaces/telescope.py
> <https://github.com/astroufsc/chimera/pull/116#discussion_r42889148>:
>
> > @@ -417,6 +417,28 @@ def unparkComplete(self):
> >          successfully.
> >          """
> >
> > +    def openCover(self):
>
> My impression is that having everything under a subclass is that you would
> only inherit to your driver the pertinent classes and then chimera would
> test it with issubclass... Although I thought this was the idea behind
> it, I never saw it implemented this way in chimera.
>
> I also remember convincing myself that this could live together with the
> "InstrumentFeatures", but I completely forgot the argument now.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/astroufsc/chimera/pull/116/files#r42889148>.
>